### PR TITLE
Fix performance issue in transaction screen.

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
I optimize logic for creating of the sections in the transaction view controller and reload of the table view.

### CODE REVIEW

**Comment by OlegGordiichuk** [comment]
@aleph7 and @michaelScoff  could you pls review logic with performance improvement.

File: Trust/Transactions/ViewModels/TransactionsViewModel.swift [hunk 1]
```diff
@@ -3,6 +3,11 @@
 import Foundation
 import UIKit
 
+struct TransactionsSectionModel {
+    var header: String
+    var items: [Transaction]
```
Comment (vikmeup) [inline Trust/Transactions/ViewModels/TransactionsViewModel.swift]: This could be `let` both of them

**Comment by vikmeup** [comment]
@OlegGordiichuk what will happen if you scroll down, you have new transaction added to the top section, when you scroll up - would it be there?

**Comment by vikmeup** [comment]
@OlegGordiichuk is there any issues with getting new transactions? what's left on this PR?

**Comment by vikmeup** [comment]
@OlegGordiichuk should we close this for now? seems like you have better solution

**Pull request closed without merging**